src/validations/access-checker/wearables.ts

```diff
@@ -66,10 +100,23 @@ const parseUrnNoFail = async (urn: string): Promise<SupportedAssets | null> => {
     if (parsed?.type === 'off-chain') {
       return parsed
     }
+    if (parsed?.type === 'blockchain-collection-third-party') {
+      return parsed
+    }
   } catch {}
   return null
 }
 
+const toHexBuffer = (value: string): Buffer => {
+  if (value.startsWith('0x')) {
+    return Buffer.from(value.substring(2), 'hex') // removing first 2 characters (0x)
+  }
+  return Buffer.from(value, 'hex')
+}
+
+const getThirdPartyId = (urn: BlockchainCollectionThirdParty): string =>
+  `urn:decentraland:${urn.network}:collections-thirdparty:${urn.thirdPartyName}`
```

Comment:
doesn't the urn parser lib expose a method to do this?

Reply:
for the time being is just plain data, no logic there.